### ISSUE TITLE
phpcs:ignore annotation doesnt work inside a docblock for properties or methods

### ISSUE THREAD

**Issue by daften** [squizlabs/PHP_CodeSniffer#3406]
**Describe the bug**

A follow-up for #2868 where the issue doesn't seem completely fixed.
When using codesniffer 3.6.0 and adding something like this:

```php
/**
   * The property.
   *
   * @var string
   *
   * @phpcs:ignore Drupal.NamingConventions.ValidVariableName.LowerCamelName
   */
  private $Property;
```

It doesn't work, the error still shows up.

**To reproduce**
1. Create a file called `test.php` with the code sample above inside a class, and using the Drupal coding standards. Or use other standards and ignore another rule to ignore.
2. Run `phpcs test.php ...`
3. See error message displayed
```
-------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------
 30 | ERROR | Class property $Property should use lowerCamel naming without underscores
    |       | (Drupal.NamingConventions.ValidVariableName.LowerCamelName)
-------------------------------------------------------------------------------------------------------------------
```

**Expected behavior**
The Drupal.NamingConventions.ValidVariableName.LowerCamelName rule should be ignored for that propery.

**Versions (please complete the following information):**
 - OS: [e.g., MacOS 11.5.1]
 - PHP: [e.g., 7.3]
 - PHPCS: [e.g., 3.6.0]
 - Standard: [Drupal]

**Comment by jrfnl** [comment]
This is working exactly as designed per the specifications:
> You can also ignore a single line using the `phpcs:ignore` comment. This comment will ignore the line that the comment is on, and the following line.

Ref: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file

If you want to ignore something on the line below the docblock, you can either use a trailing comment or use a `phpcs:disable/enable` combo.

```php
/**
 * The property.
 *
 * @var string
 */
private $Property; // phpcs:ignore Drupal.NamingConventions.ValidVariableName.LowerCamelName

/**
 * The property.
 *
 * @var string
 *
 * @phpcs:disable Drupal.NamingConventions.ValidVariableName.LowerCamelName
 */
private $Property;
// phpcs:enable
```

You may want to read up on the use of these annotations by reading the Wiki section I linked to above.

**Comment by daften** [comment]
Aaah, my apologies. I'll close this issue. I've used the disable method atm, so I'll leave that in now. Thanks for the quick replies! :)
